tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>Again here: circuit-&gt; algorithm/model</p><p> </p><p>This pargraph should really emphasize the power of high-level functional design and that the circuit implementation of each oracle model can vary substantially according to the constraints and preferences. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>